app/views/item_seasons/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>ItemSeasons#index</h1>
```

Comment:
If feels like we do not need the item_seasons views and controller. Any particular reason for us to have them already?

Reply:
nope, I thought we were going to need them, but that is not the case

Reply:
Nope, let's delete everything that we do not use.